readthedocs/search/tests/data/sphinx/in/no-title.html

```diff
@@ -1,7 +1,20 @@
+<!DOCTYPE html>
```

Comment:
I understand the HTML files has to be valid HTML now to be able to parse them, right?

Reply:
yeah, previously these were just an extract of the full document.